addons/repair/tests/test_repair.py

```diff
@@ -719,40 +665,13 @@ def test_create_repair_order_from_cross_company_sn(self):
         })
         self.assertEqual(repair_order.company_id, self.env.company)
 
-    def test_delivered_qty_of_generated_so(self):
```

Comment:
As far as I know, this flow is still supported (creating a SO from a RO), so no reason to delete this test.